scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ZIP_NAME="deployment.zip"
+
+pip install --target ./reqs -r sleuth/requirements.txt
+cd reqs
+zip -r9 ../$ZIP_NAME .
+cd ../sleuth
+zip -r9 ../$ZIP_NAME handler.py
+zip -r9 ../$ZIP_NAME -r sleuth/
+echo "Packaging complete, saving as artifact for troubleshooting"
```

Comment:
I think this is misleading.... artifact is just saved ... no need for "troubleshooting"